target/bin/ldd

```diff
@@ -24,9 +24,9 @@
 
 # We should be able to find the translation right at the beginning.
 TEXTDOMAIN=libc
-TEXTDOMAINDIR=/home/dennis/Documents/lind-wasm/glibc/target/share/locale
```

Comment:
we probably don't want absolute paths here pointing to Dennis's home folder?

Reply:
Same for a lot of these. Would be good to add comments for modified build files.

Reply:
This line is deleted, and I will comment for absolute paths